### ISSUE TITLE
Please update ONBUILD documentation

### ISSUE THREAD

**Issue by anjackson** [klakegg/docker-hugo#18]
The documentation at: https://github.com/klakegg/docker-hugo#using-an-onbuild-image appears to be out of date. As of `klakegg/hugo:0.65.3-onbuild` the name for the output folder is `/target` (not `/onbuild`). This means a working ONBUILD container looks like this:

```
FROM klakegg/hugo:0.65.3-onbuild AS hugo

FROM nginx
COPY --from=hugo /target /usr/share/nginx/html
```

**Comment by anjackson** [comment]
Ah, actually that doesn't work because `/target` is a Docker volume so it gets dropped. I couldn't get this to work with the ONBUILD image.  I think it needs changing to force `HUGO_DESTINATION=/onbuild`.

**Comment by klakegg** [comment]
Thank you for reporting this. I was able to reproduce this, and have made the necessary changes. This fix will be shipped with the 0.66.0 release.
